i18n/sample/src/api/ZoweExplorerZosmfApi.i18n.json

```diff
@@ -1,3 +1,4 @@
 {
+    "getValidSession.loginImpossible": "Profile {0} is invalid. Please check your login details and try again.",
```

Comment:
When is this message called?

Reply:
[here](https://github.com/zowe/vscode-extension-for-zowe/blob/e6631a413cfc8352963e6ec9f279d2d573838992/src/api/ZoweExplorerZosmfApi.ts#L214)